platforms/nuttx/src/bootloader/microchip/mpfs/main.c

```diff
@@ -590,81 +698,32 @@ static size_t get_image_size(void)
 }
 #endif
 
-#ifdef CONFIG_MMCSD
-void partition_handler(FAR struct partition_s *part, FAR void *arg)
-{
-	//_alert("found partition %d\n",part->index);
-	if (part->index == 0) {
-		register_blockpartition("/dev/mmcsd0p0", 0, "/dev/mmcsd0", part->firstblock, part->nblocks);
-	}
-}
-#endif
-
-static int load_sdcard_images(const char *name, uint64_t loadaddr)
-{
-	int mmcsd_fd;
-	struct stat file_stat;
-
-	_alert("Loading %s\n", name);
-	mmcsd_fd = open(name, O_RDONLY);
-
-	if (mmcsd_fd > 0) {
-		fstat(mmcsd_fd, &file_stat);
-		size_t got = read(mmcsd_fd, (void *)loadaddr, file_stat.st_size);
-
-		if (got > 0) {
-			close(mmcsd_fd);
-			return 0;
-		}
-	}
-
-	close(mmcsd_fd);
-	return -1;
-}
-
-
 static int loader_main(int argc, char *argv[])
 {
+	int ret;
 	ssize_t image_sz = 0;
 
 	loading_status = IN_PROGRESS;
 	_alert("%s %d\n", __func__, __LINE__);
-#ifdef CONFIG_MMCSD
 
+#if defined(CONFIG_MMCSD)
 
-	parse_block_partition("/dev/mmcsd0", partition_handler, "/sdcard/");
-	_alert("%s %d\n", __func__, __LINE__);
-	/*
-	 * Mount the sdcard and check if the image is present
-	 */
-	int ret = mount("/dev/mmcsd0p0", "/sdcard/", "vfat", 0, NULL);
-
-	if (ret >= 0) {
-		int mmc_fd = open("/sdcard/boot/" IMAGE_FN, O_RDWR | O_CREAT);
+	//_alert("%s %d\n", __func__, __LINE__);
```

Comment:
Personally I would prefer leaving commented code out, or at least give a reason why the code is there and why it is commented.

The informative value of this print is not immediately obvious to me, seems like something that was used/added for debugging.

Reply:
Yep, leftovers from some debug. Will remove

Reply:
Now removed